js/battle.js

```diff
@@ -2479,7 +2479,7 @@ var Battle = (function () {
 		} else {
 			this.logFrameElem.html('<div class="battle-options"></div>');
 			this.optionsElem = this.logFrameElem.children().last();
-			this.logFrameElem.append('<div class="inner"></div>');
+			this.logFrameElem.append('<div class="inner" role="log" title="Battle Log"></div>');
```

Comment:
`title` is a bad attribute; I don't want that to appear as a tooltip for sighted readers. Does there exist an attribute like `aria-title`?